sycl/source/detail/scheduler/commands.cpp

```diff
@@ -2196,6 +2196,9 @@ static pi_result SetKernelParamsAndLaunch(
 
       RT::PiMem MemArg = (RT::PiMem)getMemAllocationFunc(Req);
       if (Queue->getDeviceImplPtr()->getBackend() == backend::opencl) {
+        if (!MemArg)
+          MemArg = RT::PiMem();
```

Comment:
Please add a comment here as to why it is needed.

Reply:
thansk @steffenlarsen , done.